commit 5ee106e9ce85b06647257a3d4d147fd83264a7d2 (refs/changes/00/1700/2)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2017-03-22T16:57:23+00:00 (2 years, 7 months ago)
    
    MANATEE-324 pull MANATEE-288 fix into manta-manatee, sdc-manatee
    Reviewed by: David Pacheco <dap@joyent.com>
    Approved by: David Pacheco <dap@joyent.com>

diff --git a/package.json b/package.json
index 76c42fe..ff84678 100644
--- a/package.json
+++ b/package.json
@@ -13,7 +13,7 @@
     },
     "dependencies": {
         "carrier": "0.2.0",
-        "manatee": "git+https://github.com/joyent/manatee.git#acbcf534d671d9f75a7c3092520f650bbfb6b803"
+        "manatee": "git+https://github.com/joyent/manatee.git#86ae52d4a9988958aec3ec9955f3e2190bfa81c7"
     },
     "sdcDependencies": {
         "config-agent": ">=1.2.0"
